public/locales/en.js

```diff
@@ -324,6 +324,7 @@ export default {
         recordedAt: 'Recorded At',
         completedAt: 'Completed At',
         recordedBy: 'Recorded By',
+        recordedByDescription: 'User name that recorded this operations log',
```

Comment:
Shouldn't this be / use the language of "performed by"?

Reply:
good catch ... there are still some lingering "recorded by" references in there. Lemme add a commit to clean that up.

Reply:
fixed in [18dbd0d](https://github.com/AWeber-Imbi/imbi-ui/pull/112/commits/18dbd0df78f99b1396f3e26ac08860f4c539f4e5)